### ISSUE TITLE
TF2: Another casual bypass preload that needs patching

### ISSUE THREAD

**Issue by Eavontide** [ValveSoftware/Source-1-Games#5237]
As of today's update with the one method of bypassing sv_pure for casual being patched out, I feel it is my duty to alert you that there is another method using VScript that needs addressing.

[https://gamebanana.com/wips/79779](https://gamebanana.com/wips/79779)

**Comment by gaelcoral** [comment]
Although I enjoy seeing the playermodel changed in the Matchmacking.
Valve/janitor please fix this!
This can give an advantage to some bad actors without the need for cheats.

**Comment by BrochachoTheBro** [comment]
THIS ACCOUNT ISN'T ACTUALLY EAVE, ITS AN IMPERSONATOR (likely TF2CC or one of his cronies) TRYING TO STAIN THEM

@Raumzelleth is their actual account

**Comment by ImguRandom** [comment]
> (likely TF2CC or one of his cronies)

NAH NOT ME. I'M RIGHT HERE. ALSO I DON'T HAVE ANY CRONIES.

...WHY ARE WE YELLING?

**Comment by ImguRandom** [comment]
> likely TF2CC or one of his cronies

NAH BRO IT AIN'T ME. I'M RIGHT HERE. AND I DON'T HAVE CRONIES.

I ALSO DON'T KNOW WHAT WE'RE YELLING ABOUT.

**Comment by Raumzelleth** [comment]
Must've been the wind...

**Comment by Eavontide** [comment]
> THIS ACCOUNT ISN'T ACTUALLY EAVE, ITS AN IMPERSONATOR (likely TF2CC or one of his cronies) TRYING TO DEFAME THEM
> 
> @Raumzelleth is their actual account

nuh uh

**Comment by Raumzelleth** [comment]
> > THIS ACCOUNT ISN'T ACTUALLY EAVE, ITS AN IMPERSONATOR (likely TF2CC or one of his cronies) TRYING TO DEFAME THEM
> > @Raumzelleth is their actual account
> 
> nuh uh

why are you still trying lmao

can you just be upfront for once and tell me something? did you reply here on your new main, realize you hadn't interacted with the Source 1 issue tracker on it before, realize that looked mighty suspicious, and then immediately delete the reply?

**Comment by sapphonie** [comment]
This is bordering on insanity.

@kisak-valve, this is an obvious throwaway account, likely of someone (404UnF/TF2CutContent) who was previously wholly banned from GitHub for spamming, harassment, and other such nonsense.


Plus, this is literally not going to get fixed as doing so would require a ridiculous amount of time spent on the model loading / material loading backend of Source - basically fixing preloading entirely, which would doubtless break other things as well.

I'd recommend locking this issue and disallowing this user from participating on the Source 1 issue tracker.

Thanks
Sappho

**Comment by ImguRandom** [comment]
> did you reply here on your new main, realize you hadn't interacted with the Source 1 issue tracker on it before, realize that looked mighty suspicious, and then immediately delete the reply?

Very creative headcanon here, but no.

1. This issue report, and by extension the account that created it, was not done by me or "my cronies", of which I have none.
2. My response was a take on the Anchorman "I DON'T KNOW WHAT WE'RE YELLING ABOUT" meme but I realized it would have just been hidden as off-topic anyway, so I deleted my reply.

**Comment by Raumzelleth** [comment]
> Very creative headcanon here, but no.
> 
> 1. This issue report, and by extension the account that created it, was not done by me or "my cronies", of which I have none.
> 2. My response was a take on the Anchorman "I DON'T KNOW WHAT WE'RE YELLING ABOUT" meme but I realized it would have just been hidden as off-topic anyway, so I deleted my reply.

Fair enough. Though, never hurts to use some intuition, and at least I asked. I'd still appreciate it if you wouldn't impersonate me, though.
